python/README.md

```diff
@@ -26,57 +36,26 @@ optional arguments:
   -port PORT, --flightport PORT
                         Dremio flight server port. Defaults to 32010.
   -user USERNAME, --username USERNAME
-                        Dremio username. Defaults to "dremio".
+                        Dremio username.
   -pass PASSWORD, --password PASSWORD
-                        Dremio password. Defaults to "dremio123".
-  -pat PAT_OR_AUTH_TOKEN, --personalAccessToken PAT_OR_AUTH_TOKEN, -authToken PAT_OR_AUTH_TOKEN, --authToken PAT_OR_AUTH_TOKEN
+                        Dremio password.
+  -pat TOKEN, --token TOKEN
                         Either a Personal Access Token or an OAuth2 Token.
   -query QUERY, --sqlQuery QUERY
```

Comment:
Mention that query must be enclosed in single quotes.
Q: what happens if the query already contains single quotes?

Reply:
Good question. Based on my experience, if the query already contains single quotes, you change the existing single quotes to double quotes and enclose the query in single quotes as usual.